src/mlpack/core/dists/gamma_distribution.hpp

```diff
@@ -127,14 +127,14 @@ class GammaDistribution
              const arma::vec& meanxVec,
              const double tol = 1e-8);
 
-
   /**
    * This function returns the probability of a group of observations.
    *
    * The probability of the value x is
    *
-   * \frac{x^(\alpha - 1)}{\Gamma(\alpha) * \beta^\alpha} * e ^
-   * {-\frac{x}{\beta}}
+   * \f[
+   * \frac{x^{(\alpha - 1)}}{\Gamma(\alpha) \beta^\alpha} e^{-\frac{x}{\beta}}
+   * \f]
    *
```

Comment:
I noticed a superscript issue for the latex expression in `gamma_distribution.hpp`. 
**[Before]**
<p align=center>
<a href="https://www.codecogs.com/eqnedit.php?latex=\frac{x^(\alpha&space;-&space;1)}{\Gamma(\alpha)&space;*&space;\beta^\alpha}&space;*&space;e&space;^&space;{-\frac{x}{\beta}}" target="_blank"><img src="https://latex.codecogs.com/gif.latex?\frac{x^(\alpha&space;-&space;1)}{\Gamma(\alpha)&space;*&space;\beta^\alpha}&space;*&space;e&space;^&space;{-\frac{x}{\beta}}" title="\frac{x^(\alpha - 1)}{\Gamma(\alpha) * \beta^\alpha} * e ^ {-\frac{x}{\beta}}" /></a>
</p>

**[After]**
<p align=center>
<a href="https://www.codecogs.com/eqnedit.php?latex=\frac{x^{(\alpha&space;-&space;1)}}{\Gamma(\alpha)&space;\beta^\alpha}&space;e^{-\frac{x}{\beta}}" target="_blank"><img src="https://latex.codecogs.com/gif.latex?\frac{x^{(\alpha&space;-&space;1)}}{\Gamma(\alpha)&space;\beta^\alpha}&space;e^{-\frac{x}{\beta}}" title="\frac{x^{(\alpha - 1)}}{\Gamma(\alpha) \beta^\alpha} e^{-\frac{x}{\beta}}" /></a>
</p>

Lastly, I removed the `*` for simplicity.